packages/client/src/apis/rewards/RewardsExecuteAPI.ts

```diff
@@ -81,12 +82,11 @@ export class RewardsExecuteAPI extends BaseVertexAPI {
       return availableAmount.toFixed();
     })();
 
-    return this.context.contracts.vrtxAirdrop.claim(
-      params.epoch,
-      amountToClaim,
-      totalAmount.toFixed(),
-      proof,
-    );
+    const claimFn = params.stake
```

Comment:
Thoughts on this, versus exposing a new fn? there was a decent bit of logic up above that i wanted to share

Reply:
Fair enough